packages/account/src/Configs/financial-details-config.ts

```diff
@@ -262,28 +262,7 @@ export const getEducationLevelList = () => [
     },
 ];
 
-export const getNetIncomeList = () => [
```

Comment:
Same list of values exists, hence using that. Can add additional values in future